### ISSUE TITLE
[JS] Chapter3-Arrays -Declare an array with more than 5 number of elements

### ISSUE THREAD

**Issue by SJ-Kumar** [SJ-Kumar/hacktoberfest-2022#1]

**Comment by Shivashish-rwt** [comment]
Please assign this to me.

**Comment by SJ-Kumar** [comment]
Yeah. I have assigned it to you. Looking forward to your pull request.

**Comment by SJ-Kumar** [comment]
> Just wanna know where should i put my file??

## After you complete any of the selected excercise, you can create a folder with your name at the same level and commit.

**Comment by Shivashish-rwt** [comment]
Hey @SJ-Kumar!!
I have created a pull request. Please review it and if u want any changes let me know.

**Comment by SJ-Kumar** [comment]
> Hey @SJ-Kumar!! I have created a pull request. Please review it and if u want any changes let me know.

I am unable to see your file. Please confirm if you have raised a pull request.
